sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<h2>Create an XGBoost Estimator</h2><p>Here we create a SageMaker Estimator using the XGBoost Image. We attach the SageMaker Debugger built-in <code><a href="https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-training-xgboost-report.html" target="_blank">create_xgboost_report()</a></code> rule to automatically generate an XGBoost training report after the training job is complete. SageMaker Debugger also turns on the <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-profiling-report.html" target="_blank">ProfilerReport</a> rule and autogenerate a report regarding system resource utilization and bottleneck detection results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done